src/utils/isSameYearAndMonth.test.ts

```diff
@@ -1,12 +1,12 @@
-import { isSameMonth } from 'date-fns'
+import isSameYearAndMonth from './isSameYearAndMonth'
```

Comment:
The `date-fns library` was being tested.

Reply:
cool